src/renderer/shaders/styler/stylerEncodeWidth.glsl

```diff
@@ -1,4 +1,10 @@
 vec2 encodeWidth(float x) {
+    if (!(x<=0. || x>=0.) ){
```

Comment:
Is `isnan` not always available?

Reply:
In WebGL 1 is not available ever, in modern GLSL it is: https://stackoverflow.com/questions/11810158/how-to-deal-with-nan-or-inf-in-opengl-es-2-0-shaders